utils/amt-integration-tests/tests/db_prep.py

```diff
@@ -17,16 +17,27 @@ def _sqlcmd_parameters_from(config: ServerConfig) -> List[str]:
         else ["-U", config.username, "-P", config.password]
     )
 
-    return ["-d", config.db_name, "-S", f"{config.server},{config.port}", *login]
+    ENABLE_QUOTED_IDENTIFIERS = "-I"
```

Comment:
Interestingly, quoted identifiers are ON by default with SSMS, but OFF by default with `sqlcmd.exe`. Without this flag I was getting an error when running one of our scripts that creates new indexes.